web/src/main/webapp/WEB-INF/jsp/managestudy/studySubject/casebookGenerationForm.jsp

```diff
@@ -6,54 +6,87 @@
-                    <input type="checkbox" checked="" value="includeDNs=y" id="casebookParamDn" name="casebookParam">
-                    <fmt:message key="viewStudySubject.casebookGenerationForm.option.dn" bundle="${resword}"/>
-                    <input type="checkbox" checked="" value="includeAudits=y" id="casebookParamAudit" name="casebookParam">
-                    <fmt:message key="viewStudySubject.casebookGenerationForm.option.auditTrail" bundle="${resword}"/>
-                    <input type="checkbox" checked="" value="includeMetadata=y" anti-value="includeMetadata=n" id="casebookParamMetadata" name="casebookParam">
-                    <fmt:message key="viewStudySubject.casebookGenerationForm.option.includeMetadata" bundle="${resword}"/>
-                    <input type="checkbox" checked="" value="showArchived=y" anti-value="showArchived=n" id="casebookParamArchived" name="casebookParam">
-                    <fmt:message key="viewStudySubject.casebookGenerationForm.option.includeArchived" bundle="${resword}"/>
-                </div>
+            <div style="margin-bottom:10px;" class="connect-widget">
 
-                <div style="margin-bottom:10px;" class="connect-widget">
-
-                    <div id="casebookButtons">
-                        <input type="button" id="casebookLinkBtn" class="button_medium"
-                               value="<fmt:message key="viewStudySubject.casebookGenerationForm.button.getLink" bundle="${resword}"/>"
-                               name="GetLink">
-                        <input type="button" id="casebookOpenBtn" class="button_medium"
-                               value="<fmt:message key="viewStudySubject.casebookGenerationForm.button.open" bundle="${resword}"/>"
-                               name="exit">
-                    </div>
-                    <div style="display:none" id="casebookLinkDisplay"><br>
-                        <fmt:message key="viewStudySubject.casebookGenerationForm.button.getLink.description"
-                                     bundle="${resword}"/><br>
-                        <input type="text" id="casebookLinkText" size="100"></div>
+                <div id="casebookButtons">
+                    <input type="button" id="casebookLinkBtn" class="button_medium"
+                           value="<fmt:message key="viewStudySubject.casebookGenerationForm.button.getLink" bundle="${resword}"/>"
+                           name="GetLink">
+                    <input type="button" id="casebookOpenBtn" class="button_medium"
+                           value="<fmt:message key="viewStudySubject.casebookGenerationForm.button.open" bundle="${resword}"/>"
+                           name="exit">
+                    <input type="button" id="casebookPdfBtn" class="button_medium"
+                           value="<fmt:message key="viewStudySubject.casebookGenerationForm.button.getPdf" bundle="${resword}"/>"
```

Comment:
This button should make an ajax call to the printable casebook API. This API is still being developed.

Reply:
OK ajax called (but it was still 404 when I tested)